src/quo2/components/numbered_keyboard/keyboard_key/view.cljs

```diff
@@ -0,0 +1,34 @@
+(ns quo2.components.numbered-keyboard.keyboard-key.view
+  (:require [react-native.core :as rn]
+            [quo2.theme :as quo.theme]
+            [status-im.ui.components.icons.icons :as icons]
+            [quo2.foundations.colors :as colors]
+            [quo2.components.numbered-keyboard.keyboard-key.style :as style]))
+
+(defn- get-label-color
+  [disabled theme blur]
+  (if disabled
+    (if (or (= :dark theme) blur)
+      colors/white-opa-30
+      colors/neutral-30)
+    (if (or (= :dark theme) blur)
+      colors/white
+      colors/neutral-100)))
+
+(defn- keyboard-key-internal
```

Comment:
for labels we have a common pattern as using the second parameter as the label.
i.e
```
(defn- keyboard-key-internal
  [{:keys [ disabled theme blur? icon? on-press?]} label]
  ```
  
  I think we should try to stick with Figma here (even if it's a bit much in this case - it will align with best practices then)
  e.g
  ```
 
(defn- keyboard-key-internal
  [{:keys [disabled theme blur? icon? on-press? type]} label]
  (let [label-color (get-label-color disabled theme blur?)]
    [rn/touchable-highlight
     {:active-opacity 1
      :underlay-color (cond
                        blur?            colors/white-opa-10
                        (= :light theme) colors/neutral-10
                        (= :dark theme)  colors/neutral-80)
      :disabled       disabled
      :on-press       on-press?
      :style          style/container}
     (cond 
       (= type :key)  [icons/icon icon? (style/icon label-color)]
       (= type :digit) [rn/text {:style (style/text label-color)} label])]))
 ```

Reply:
Done